health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L1Main.js

```
@@ -0,0 +1,689 @@
+      icon: "PDFSvg",
+      label: t("ES_DSS_DOWNLOAD_PDF"),
+      code: "ES_DSS_DOWNLOAD_PDF",
+    },
+  ];
+
+  const onActionSelect = (item) => {
+    switch (item?.code) {
+      case "ES_DSS_DOWNLOAD_IMAGE":
+        setTimeout(() => {
+          return Digit.Download.Image(fullPageRef, t(dashboardConfig?.[0]?.name));
+        }, 500);
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)